test/e2e/tests/ratelimit.go

```diff
@@ -35,7 +35,7 @@ var RateLimitTest = suite.ConformanceTest{
 
 			// TODO: find a better to make sure ratelimit load new configuration
```

Comment:
we delete/update this lines now?

Reply:
we should keep that line until we delete the sleep with something more reliable, like some xds confirmation in route status

Reply:
I am working on delete this two TODO lines right now